configurations/manager/100GB_dataset.yaml

```diff
@@ -5,10 +5,8 @@ prepare_write_cmd: [ "cassandra-stress write cl=ALL n=26214400 -schema 'replicat
                      "cassandra-stress write cl=ALL n=26214400 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native  -rate threads=500 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=52428801..78643200",
                      "cassandra-stress write cl=ALL n=26214400 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native  -rate threads=500 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=78643201..104857600" ]
 
-stress_read_cmd: [ "cassandra-stress read cl=ALL n=26214400 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native  -rate threads=50 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..26214400",
-                   "cassandra-stress read cl=ALL n=26214400 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native  -rate threads=50 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=26214401..52428800",
-                   "cassandra-stress read cl=ALL n=26214400 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native  -rate threads=50 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=52428801..78643200",
-                   "cassandra-stress read cl=ALL n=26214400 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native  -rate threads=50 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=78643201..104857600" ]
+stress_cmd: [ "cassandra-stress read  cl=ALL duration=10m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native  -rate threads=100 -col 'size=FIXED(1024) n=FIXED(1)'",
+              "cassandra-stress write cl=ALL duration=10m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native  -rate threads=100 -col 'size=FIXED(1024) n=FIXED(1)'", ]
```

Comment:
If you want to measure latency of these operations, you need to throttle the throughput by using:
" -rate threads= x **fixed=YY/s**" 

The value of "fixed" is the "throughput / num of loaders" the test is targeting.
We usually set it to 50% of max throughput.
The target of "cpu utilization" is irrelevant and shouldn't be part of the test consideration.

Reply:
Why are you using cl=ALL?
Why are you using 2 separate commands and not one stress commands that is doing "mixed" workload.

Reply:
The command doesn't define the distribution, better to state it even if it uses the default "seq".

Reply:
> The target of "cpu utilization" is irrelevant and shouldn't be part of the test consideration.

Irrelevant to what/who? and why?
When sizung and tuning clusters in the field do you think tgat cystomers care much about sone benchmark?
That can run their app or simulated workload tgat represent it and tune the system to certain utilizantion level (cpu, network, disk throughput, erc.) to leave enough headroom for maintenance load (compaction, repair, backup, etc.(
But if in QA, the bar is currently set at 50% of the measured throughput, that's fine too for testing consistency purposes.

Reply:
The CPU utilization is irrelevant exactly because you don't know what this CPU means to the customer.
10% of it could be serving statements and the rest is doing compactions, streaming handling the backup or whatever.
You don't set your bar because this is your comfort zone, you set the bar at something meaningful to the customer, e.g. 50% of throughput.

Reply:
done